src/session-managers/SecureSessionManager.ts

```diff
@@ -1,8 +1,11 @@
 /* eslint-disable @typescript-eslint/no-unsafe-argument */
-/// <reference types="@fastify/secure-session" />
 import { FastifyRequest } from 'fastify'
 import { AuthenticateOptions } from '../AuthenticationRoute'
 import { SerializeFunction } from '../Authenticator'
+import { FastifySessionObject } from '@fastify/session'
+import { Session } from '@fastify/secure-session'
+
+type Request = FastifyRequest & { session: FastifySessionObject | Session<any> }
```

Comment:
Can we use `SessionData` in place of `any`? [ref](https://github.com/fastify/fastify-secure-session/blob/master/types/index.d.ts#L27) 
```suggestion
type Request = FastifyRequest & { session: FastifySessionObject | Session<SessionData> }
```